src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1094,5 +1094,10 @@ configurationRegistry.registerConfiguration({
 			type: 'boolean',
 			default: true
 		},
+		[NotebookSetting.outputBackupSizeLimit]: {
+			markdownDescription: nls.localize('notebook.backup.sizeLimit', "The limit of notebook output size where notebook files will no longer be backed up for hot reload. Use 0 for unlimited."),
+			type: 'number',
+			default: 10000
```

Comment:
Is this value based on some measurements? It seems quite low, for example Luna Paint picked 100kb.

Reply:
this is in Kilobytes, so 10 mb (that used to be in the description, I will add it back in)

Reply:
Hmm, I wonder if we should be consistent here with other settings and pick `B` instead?

Reply:
maybe MB should be used? workbench.editorLargeFileConfirmation also does

Reply:
Oh yeah true, I was wrong, even the local history setting uses KB:

<img width="1062" alt="image" src="https://github.com/microsoft/vscode/assets/900690/bcdb3499-d5d1-453d-a428-9194ef63440f">